inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -175,7 +175,7 @@
                     },
                     "full_name": {
                         "description": "author name as it appears in the paper",
-                        "format": ".+, .+",
+                        "pattern": ".+, .+",
```

Comment:
Actually, the regexp was already there, I assumed all existing people would match the regexp. So, since we have Jimmy and possible others like him, it may be better idea remove this validation.